miprometheus/models/encoder_solver/maes_model.py

```diff
@@ -100,24 +100,33 @@ def __init__(self, params, problem_default_values_={}):
 
 
 
-    def save(self, model_dir, stat_col):
+    def save(self, model_dir, training_status, training_stats, validation_stats):
         """
-        Method saves the model and encoder to file.
+        Generic method saving the model parameters to file. It can be \
+        overloaded if one needs more control.
 
         :param model_dir: Directory where the model will be saved.
-        
-        :param stat_col: Statistics collector that contain current loss and episode number (and other statistics).
+        :type model_dir: str
+
+        :param training_status: String representing the current status of training.
+        :type training_status: str
+
+        :param training_stats: Training statistics that will be saved to checkpoint along with the model.
+        :type training_stats: :py:class:miprometheus.utils.StatisticsAggregator or :py:class:miprometheus.utils.StatisticsAggregator
+
```

Comment:
Small thing: you are missing back ticks and you copy-pasted twice the same class name ;)

Reply:
done :)